messaging/src/test/java/org/axonframework/eventhandling/scheduling/jobrunr/JobRunrScheduleTokenTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.eventhandling.scheduling.jobrunr;
+
+import org.axonframework.eventhandling.scheduling.ScheduleToken;
+import org.junit.jupiter.api.*;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JobRunrScheduleTokenTest {
+
```

Comment:
I think it's good to add some serialization tests for the token too, that validate both the XStream and Jackson serializer.

Reply:
Added